src/test/regress/expected/background_task_queue_monitor.out

```diff
@@ -302,6 +308,195 @@ SELECT job_id, task_id, status FROM pg_dist_background_task
+      17 | cancelled
+      18 | cancelled
+      19 | cancelled
+      20 | cancelled
+(5 rows)
+
+-- verify that upon termination signal, all tasks fail and retry policy sets their status back to runnable
+BEGIN;
+INSERT INTO pg_dist_background_job (job_type, description) VALUES ('test_job', 'simple test to verify max parallel background execution') RETURNING job_id AS job_id1 \gset
+INSERT INTO pg_dist_background_task (job_id, command) VALUES (:job_id1, $job$ SELECT pg_sleep(500); $job$) RETURNING task_id AS task_id1 \gset
+INSERT INTO pg_dist_background_job (job_type, description) VALUES ('test_job', 'simple test to verify max parallel background execution') RETURNING job_id AS job_id2 \gset
+INSERT INTO pg_dist_background_task (job_id, command) VALUES (:job_id2, $job$ SELECT pg_sleep(500); $job$) RETURNING task_id AS task_id2 \gset
+COMMIT;
+SELECT citus_job_wait(:job_id1, desired_status => 'running');
+ citus_job_wait
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT citus_job_wait(:job_id2, desired_status => 'running');
+ citus_job_wait
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT task_id, status FROM pg_dist_background_task
+    WHERE task_id IN (:task_id1, :task_id2)
+    ORDER BY task_id;
+ task_id | status
+---------------------------------------------------------------------
+      21 | running
+      22 | running
+(2 rows)
+
+SELECT pid AS monitor_pid FROM pg_stat_activity WHERE application_name ~ 'task queue monitor' \gset
+SELECT pg_terminate_backend(:monitor_pid); -- terminate monitor process
+ pg_terminate_backend
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT pg_sleep(2); -- wait enough to show that tasks are terminated
+ pg_sleep
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT task_id, status, retry_count, message FROM pg_dist_background_task
+    WHERE task_id IN (:task_id1, :task_id2)
+    ORDER BY task_id; -- show that all tasks are runnable by retry policy after termination signal
+ task_id |  status  | retry_count |                                                            message
+---------------------------------------------------------------------
+      21 | runnable |           1 | FATAL: terminating background worker "Citus Background Task Queue Executor: regression/postgres" due to administrator command+
+         |          |             | CONTEXT: Citus Background Task Queue Executor: regression/postgres                                                           +
+         |          |             |
+      22 | runnable |           1 | FATAL: terminating background worker "Citus Background Task Queue Executor: regression/postgres" due to administrator command+
+         |          |             | CONTEXT: Citus Background Task Queue Executor: regression/postgres                                                           +
```

Comment:
sidenote, Now that we can have multiple concurrent background task executors, I think the background the worker names should include the task_id. This can be useful for debugging.